fontforge/splineutil2.c

```diff
@@ -2334,7 +2334,8 @@ return( true );
 return( false );
 }
 
-SplineSet *SplineSetJoin(SplineSet *start,int doall,real fudge,int *changed) {
+SplineSet *_SplineSetJoin(SplineSet *start,int doall,real fudge,int *changed,
+                          int doLoops) {
```

Comment:
nit - stylistically, this should be lower/snake case

Reply:
fixed